src/main/kotlin/no/nav/familie/tilbake/service/dokumentbestilling/brevmaler/Dokumentmalstype.kt

```diff
@@ -0,0 +1,9 @@
+package no.nav.familie.tilbake.service.dokumentbestilling.brevmaler
+
+
+enum class Dokumentmalstype(val kode: String, val navn: String) {
+    INNHENT_DOKUMENTASJON("INNHEN", "Innhent dokumentasjon"),
```

Comment:
Trengs kode? Kode og navn er vel berre arv frå kodeverk/kodeliste i fptilbake.

Trengs navn? Er vel frontend som har ansvar for å vise riktig tekst?